src/app/modules/billing/__tests__/billing.factory.spec.ts

```diff
@@ -39,6 +45,18 @@ describe('billing.factory', () => {
         expect(actualResults._unsafeUnwrap()).toEqual([])
       })
     })
+
+    describe('addLogin', () => {
+      it('should return MissingFeatureError', async () => {
```

Comment:
nitpick - `it` statements should explain the context that they are testing e.g. "it should return MissingFeatureError if provided with an invalid form object".

that said, should this particular test be left to the type system instead?

Reply:
we could also take this opportunity to rename the `addLogin` function for better clarity e.g. `recordLoginByForm`

Reply:
>nitpick - it statements should explain the context that they are testing e.g. "it should return MissingFeatureError if provided with an invalid form object".

this test actually doesn't require any qualifications, as it is testing that the function _always_ returns `MissingFeatureError`. if you look at the context of the nesting `describe` block, this test is for the case when `spcp-myinfo feature disabled`.

> that said, should this particular test be left to the type system instead?

could you explain what you mean by this? factories are always types to return either the service function (if the feature is enabled) or a `MissingFeatureError`/passthrough function (if the feature is disabled). this test is checking that the factory correctly provides a function which only returns a `MissingFeatureError` if the `spcp-myinfo` feature is disabled.